### PR TITLE
Mention jq as a tool for accessing env vars on the command line.

### PR DESCRIPTION
**Pull request by Crell**

### CODE REVIEW

File: src/development/variables.md [hunk 1]
```diff
@@ -232,12 +232,17 @@ While the same command on the `feature-x` branch would produce:
...
 * [Node.js: the process.env object](https://nodejs.org/api/process.html#process_process_env)
 * [Python: the os.environ object](https://docs.python.org/3/library/os.html#os.environ)
 * [Ruby: the ENV accessor](https://ruby-doc.org/core-2.1.4/ENV.html)
 
 
+{% codetabs name="Shell", type="bash" -%}
+
+export DB_USER= echo $PLATFORM_RELATIONSHIPS | base64 --decode | jq -r ".database[0].username"
+
 {% codetabs name="PHP", type="php" -%}
```
Comment (chadwcarlson) [inline src/development/variables.md:246]:
```suggestion
{% language="PHP", type="php" -%}
```

Comment (chadwcarlson) [reply src/development/variables.md:246]: @Crell Maybe this is the issue here? `codetab` only  on the first tab?

File: src/development/variables.md [hunk 2]
```diff
@@ -232,13 +232,18 @@ While the same command on the `feature-x` branch would produce:
...
 * [PHP: the getenv() function](http://php.net/manual/en/function.getenv.php)
 * [Node.js: the process.env object](https://nodejs.org/api/process.html#process_process_env)
 * [Python: the os.environ object](https://docs.python.org/3/library/os.html#os.environ)
 * [Ruby: the ENV accessor](https://ruby-doc.org/core-2.1.4/ENV.html)
 
 
-{% codetabs name="PHP", type="php" -%}
+{% codetabs name="Shell", type="bash" -%}
+
+export DB_USER=echo $PLATFORM_RELATIONSHIPS | base64 --decode | jq -r ".database[0].username"
```
Comment (pjcdawkins) [inline src/development/variables.md:244]:
```suggestion
export DB_USER="$(echo "$PLATFORM_RELATIONSHIPS" | base64 --decode | jq -r '.database[0].username')"
```

File: src/development/variables.md [hunk 3]
```diff
@@ -304,7 +309,7 @@ Certain variable name prefixes have special meaning.  A few of these are defined
 
 ### Top-level environment variables
 
-By default, project and environment variables are only added as part of the `$PLATFORM_VARIABLES` Unix environment variable.  However, you can also expose a variable as its own Unix environment variable by giving it the prefix `env:`.  
+By default, project and environment variables are only added as part of the `$PLATFORM_VARIABLES` Unix environment variable.  However, you can also expose a variable as its own Unix environment variable by giving it the prefix `env:`.
```
Comment (otaviojava) [inline src/development/variables.md:312]: Please, don't forget to mention that will became uppercase if they put the prefix `env:`

Comment (Crell) [reply src/development/variables.md:312]: That should be done but it's a separate fix from this one.  (This is just a whitespace fixup from my IDE, not part of this patch.)  Should we include it in this PR or make it separate?

**Pull request merged**